lib/engine/game/g_1860/game.rb

```diff
@@ -1590,17 +1572,11 @@ def check_connected(route, token)
             end
           end
 
-          paths_ = route.paths.uniq
-          token = blocked if blocked
-
-          flipped_to_neutral
-          if token.select(paths_, corporation: route.corporation).size != paths_.size
-            neutral_to_flipped
+          # no need to check whether cities are tokened out because of the above
+          unless route.ordered_paths.each_cons(2).all? { |pair| pair[0].connects_to?(pair[1], nil) }
```

Comment:
a, b

Reply:
OK

Reply:
Actually, this will be cleaned up the same as 18USA